pkg/deploy/defaults.go

```diff
@@ -139,6 +148,51 @@ func DefaultDevfileRegistryImage(cr *orgv1.CheCluster, cheFlavor string) string
 	}
 }
 
+func DefaultCheWorkspacePluginBrokerInitImage(cr *orgv1.CheCluster, cheFlavor string) string {
+	if cheFlavor == "codeready" {
+		// In the CRW case, we should always set the plugin broker image in the Che config map
+		return patchDefaultImageName(cr, defaultCheWorkspacePluginBrokerInitImage)
```

Comment:
There could be `IsAirGapMode` in CRW as well, couldn't it?

Reply:
`IsAirGapMode()` is also tested inside the `patchDefaultImageName` method

The difference is that we patch the image name *in all cases* for CRW and *only in case of airgap mode* for upstream Che

Reply:
Will that method `patchDefaultImageName()` be run after `DefaultCheWorkspacePluginBrokerInitImage()` called?

Reply:
it is called by it :-)

Reply:
thank you for explanation